.eslintrc.json

```diff
@@ -202,7 +202,6 @@
             "error",
             "always"
         ],
-        "strict": "error",
```

Comment:
Yeap, I extracted `MemoryFileSystemError` to a standalone file. So add "use strict" for it's classes declaration, otherwise it will break in node v4.